### PR TITLE
Implement users

### PR DESCRIPTION
**Pull request by davidpeterko**

### CODE REVIEW

File: .idea/.idea.Snackr/.idea/contentModel.xml
Comment (dougkent) [inline .idea/.idea.Snackr/.idea/contentModel.xml]: You might want to add the .idea folder to your gitignore file so that it doesn't get committed to git hub since it's just a folder that rider uses.

File: Snackr/Controllers/AccountController.cs
Comment (dougkent) [inline Snackr/Controllers/AccountController.cs:26]: Dependency Injection 👍

File: Snackr/Views/Account/Claims.cshtml
Comment (dougkent) [inline Snackr/Views/Account/Claims.cshtml:8]: What are you going to use this page for? If nothing I would remove it.

File: Snackr/Views/Account/Index.cshtml
Comment (dougkent) [inline Snackr/Views/Account/Index.cshtml]: Looks like this is just a blank page.

File: Snackr/Views/Account/Profile.cshtml
Comment (dougkent) [inline Snackr/Views/Account/Profile.cshtml:29]: Everyone has normal snack permissions? What the hell, I invited you to my wedding and I still only get normal snack permissions?

File: Snackr/Views/Home/Index.cshtml
Comment (dougkent) [inline Snackr/Views/Home/Index.cshtml]: Should be div rather than divo

File: Snackr/Views/Shared/_Layout.cshtml
Comment (dougkent) [inline Snackr/Views/Shared/_Layout.cshtml]: Remove commented code if you aren't going to use it

**Review by dougkent — CHANGES_REQUESTED** [review]
Just one major thing (Make me a snack admin!) but the rest are minor things. Over all it's looking good. 

My opinion you should add more layers and do more of an angular/web api setup for the front end rather than old Razer MVC.

File: Snackr/Views/Account/Claims.cshtml
Comment (davidkogxg) [reply Snackr/Views/Account/Claims.cshtml:8]: it was used for debugging. to pull the email, name, and etc out of the Auth0 'claims list' i just output it to a page so i could view what it was in the claimbag. i will remove it in the future when im completely satisfied with the way login works

File: Snackr/Views/Account/Index.cshtml
Comment (davidkogxg) [reply Snackr/Views/Account/Index.cshtml]: removed.

File: Snackr/Views/Account/Profile.cshtml
Comment (davidkogxg) [reply Snackr/Views/Account/Profile.cshtml:29]: hahah. i will be updating users permissions and user stuff in the next branch.

File: Snackr/Views/Home/Index.cshtml
Comment (davidkogxg) [reply Snackr/Views/Home/Index.cshtml]: fixed.

File: Snackr/Views/Shared/_Layout.cshtml
Comment (davidkogxg) [reply Snackr/Views/Shared/_Layout.cshtml]: removed.

**Comment by davidpeterko** [comment]
the requested changes were made. waiting on PR approval.

**Pull request merged**
